UPDATE: Configure Dependabot security updates on your GitHub repo

a-a-ron opened the pull request:
The purpose of this PR is to see how we can increase the completion rate and improve the user experience. Below is the August data for this module:

| Module | Completion Rate | # completed / # started |
|--------|--------|--------|
|[Configure Dependabot security updates on your GitHub repo](https://learn.microsoft.com/en-us/training/modules/configure-dependabot-security-updates-on-github-repo/?ns-enrollment-type=Collection&ns-enrollment-id=rqymc6yw8q5rey)| 16.57% | 28/169 |

**For reference:** See the [parent issue](https://github.com/githubpartners/microsoft-learn/issues/35) for context

## Module file structure
![Screen Shot 2022-09-28 at 2 09 35 PM](https://user-images.githubusercontent.com/6351798/192878879-42d3c3d2-3ce0-4bb7-b634-8268b56f6356.png)

## Items to review for this module

We need to conduct a performance review on each of these modules to see how we can increase the completion rates, improve the user experience, and maintain the level of quality that we set for our learners. 

### Items for consideration
- **Thoroughly review the module content for flow, narrative, and length**
  - Review use of images, tables, and other visual elements
  - Does the content read like a doc or like a conversation guide?
  - Does the estimated time to complete each unit remain accurate?
- **Review the introduction unit** 
  - Does it capture your attention?
  - Does it contain repetitive content from the course title page?
  - Do the learning objectives/scope align with the content? 
  - Does the target audience match the content?
  - Do we accurately describe the module and learning objectives? (Is it possible that learners start the course thinking it's one thing to find out it's another?) 
- **Review the module exercise**
  - Does it work?
  - Does it enforce a learning objective?
  - How is it placed in the module? Should we move it around?
  - Do we need other exercises?
- **Review the questions at the end of the module**
  - Do the questions test on content taught in the module?
  - Do the questions follow best practices for item writing?

## Estimated Work Effort
| Task | Hours | 
|--------|--------|
| Detailed module review | 18 hours | 
| Action items as identified from review | TBD |

a-a-ron committed:
Update 1-introduction.yml

camihmerhar committed:
Update 1-introduction.md

There seemed to be a lot of learning objectives and to me it seemed overwhelming so I tried to pare them down and changed them to reflect the unit titles to further provide insight to the learner.

camihmerhar committed:
Update 2-manage-your-dependencies-github.md

When I reread this section I felt myself a bit confused by the unit name and why it didn't outwardly have something to do with Dependabot. So I retooled the dependabot section to come first that way we introduce how the dependency graph and the GitHub Advisory Database build into Dependabot.

camihmerhar commented:
### Action Items For Each Module
- **Thoroughly review the module content for flow, narrative, and length**
  - Review use of images, tables, and other visual elements
    **_- Yes, to me they do._**
  - Does the content read like a doc or like a conversation guide?
    **_- I would say more like a conversation guide, but because this unit has a lot of subsections with steps to enable certain settings it does read like a manual from time to time but I don’t think it deters in those sections from learners reading it, but it makes it easier for them._** 
- **Review the introduction unit** 
  - Does it capture your attention? 
    **_- I believe so. I think mentioning security in the opening sentence of the introduction page helps draw learners in._** 
  - Do the learning objectives/scope align with the content? 
    **_- From my perspective they did not. I think it might be worth having a conversation about how the learning objectives are meant to interact with the reader. As they are right now, the learning objectives for me at least, took a long time to digest and only after walking through the entire module could I say yes I got that information from this content. For me at least I think the learning objectives do 2 things. 1) Takeaways the learner will have after going through the module. But more importantly 2) prepare the learner for what exactly is to come in the module._** 
  - Does the target audience match the content?
    **_- I believe so. From my perspective this seems like a high level summary of what someone should know about Dependabot and its security functionality._** 
  - Do we accurately describe the module and learning objectives? (Is it possible that learners start the course thinking it's one thing to find out it's another?) 
    **_- I think we were a bit off with the learning objectives, but hopefully that will change with the updated ones_**
- **Review the module exercise**
  - Does it work?
    **_- I might be doing something incorrect, but I wasn’t sure what the instructions were post duplicating the repo._** 
  - Does it enforce a learning objective?
     **_- Maybe?_**
  - How is it placed in the module? Should we move it around?
     **_- I think it makes sense where it is currently._**
  - Do we need other exercises?
     - **_I don’t believe so._**  
- **Review the questions at the end of the module**
  - Do the questions test on content taught in the module?
    **_- Yes_**
  - Do the questions follow best practices for item writing?
    **_- I believe so._**

camihmerhar committed:
Update index.yml

Updated learning objectives to be consistent with the updated ones in the Introduction unit

camihmerhar committed:
Update 1-introduction.md

Added a period between "enable," and "view,".

camihmerhar committed:
Update 6-exercise.md

Updated this to reflect the new exercise David created. I also copied the content from one of our higher completed modules, [Create and publish a custom GitHub actions](https://learn.microsoft.com/en-us/training/modules/create-custom-github-actions/exercise-create-custom-action) -- Let me know your thoughts!
